core/config/v2/docs/core.toml

```
@@ -210,6 +212,8 @@ KeyPath = '/home/$USER/.chainlink/tls/server.key' # Example
 HTTPSPort = 6689 # Default
 # ForceRedirect forces TLS redirect for unencrypted connections.
 ForceRedirect = false # Default
+# ListenIP specifies the IP to bind the HTTPS server to
+ListenIP = '0.0.0.0' # Default
```

I see that we have precendent for `ListenIP` in `P2P.V1`, but it is paired with `ListenPort`. Did you consider `HTTPIP`/`HTTPSIP`?
```suggestion
HTTPSIP = '0.0.0.0' # Default
```